assets/css/src/content.css

```diff
@@ -171,15 +171,6 @@
 	margin-right: auto;
 }
 
-.alignwide {
```

Comment:
Don't remove L174-182 as we need to keep at least one instance of `. alignwide` and `.alignfull`. :)

Reply:
You might have removed both already in https://github.com/wprig/wprig/pull/406.
When this PR is updated after the merge, we need to make sure `.alignwide` and `.alignfull` are added back in.